dts/bindings/wifi/infineon,airoc-wifi-spi.yaml

```diff
@@ -0,0 +1,37 @@
+description: |
+  AIROC Wi-Fi Connectivity over SPI.
+
+compatible: "infineon,airoc-wifi"
+
+include: [spi-device.yaml, "infineon,airoc-wifi-common.yaml"]
+
+properties:
+  wifi-host-wake-gpios:
```

Comment:
duplicated from common.

Reply:
True.  Turns out I can just add the "required" attribute in infineon,airoc-wifi-spi.yaml.